lib/internal/crypto/diffiehellman.js

```diff
@@ -192,6 +181,28 @@ ECDH.prototype.generateKeys = function generateKeys(encoding, format) {
 };
 
 ECDH.prototype.getPublicKey = function getPublicKey(encoding, format) {
+  var f = getFormat(format);
+  var key = this._handle.getPublicKey(f);
```

Comment:
Is there a reason not to use `const` here? In general, `let` is preferred over `var` and `const` over `let` if possible.